doc/manual/manual.tex

```diff
@@ -2268,10 +2268,152 @@ \subsection{Particles}
+\end{enumerate}
+
+\subsection{Docker Container}
+
+\subsubsection{Installing Docker and downloading the \aspect{} image}
+
+Docker is a lightweight virtualization software that allows to ship applications with all their dependencies in a simple way. It is outside of the scope of this manual to explain all possible applications of Docker, and we refer to the introduction (\url{https://www.docker.com/what-docker}) and installation and quickstart guides (\url{https://www.docker.com/products/docker}) on the Docker website for more detailed descriptions of how to set up and use the docker engine. More importantly Docker provides a marketplace for exchanging prepared docker images (called Docker Hub). After setting up the docker engine downloading a precompiled \aspect{} image from Docker Hub is as simple as typing in a terminal:
+
+\begin{verbatim}
+docker pull gassmoeller/aspect
+\end{verbatim}
+
+\subsubsection{Running \aspect{} models}
+Although it is possible to use the downloaded \aspect{} docker image in a number of different ways, we recommend the following workflow:
+
+\begin{enumerate}
+\item Create your \aspect{} input file in a folder of your choice (possibly also containing any input data that is required by your model) and navigate in a terminal into that directory.
+\item Run the docker image and mount the current directory as a read-only volume into the docker container\footnote{Note that it is possible to mount a directory as writeable into the container. However it is often associated with file permission conflicts between the host system and the container. Therefore, we recommend this slightly more cumbersome, but also more reliable workflow.}. 
+Make sure your parameter file specifies a model output directory \textbf{other} than the input directory, e.g. \texttt{/home/dealii/aspect/model\_output}. When you have started the container run the aspect model inside the container. Note that there are two \aspect{} executables in the work directory of the container: texttt{aspect} and \texttt{aspect-release}. For a discussion of the different version see Section~\ref{sec:debug-mode}, in essence: You should run texttt{aspect} first to check your model for errors, then run \texttt{aspect-release} for a faster model run.
+
+\begin{verbatim}
+docker run -it -v "$(pwd):/home/dealii/aspect/model_input:ro" \
```

Comment:
This starts a bit unmotivated. Maybe prefix it with "To sum up, the steps you will want to execute are:..."

Also, this is in essence a shell script. Instead of `verbatim`, use the `lstslisting` environment and mark it as a `ksh` script -- other examples of this abound in the manual.